### DISCUSSION TITLE
searxng-src does not appear to be a Python project

### DISCUSSION THREAD

**Discussion by egypsiano** [searxng/searxng#1638] [Q&A] [answered]
hi all
I used this instruction (https://docs.searxng.org/admin/installation-searxng.html) to use searxng on Ubuntu 20.04 as LXC on Proxmox
now i am stuck with this step _(searx-pyenv) searxng@SearXNG:~/searxng-src$ pip install -e ._
shows me this error msg:
ERROR: file:///usr/local/searxng/searxng-src does not appear to be a Python project: neither 'setup.py' nor 'pyproject.toml' found.

I tried to show list of content but i couldn't so i typed Exit  then made the following
cd /usr/local/searxng/searxng
ls
there's nothing
root@SearXNG:/usr/local/searxng# ls
' '   searx-pyenv   searxng-docker   searxng-src
what should I do?
I used this script because I have any idea about docker 
so any one can advise me, and please consider that I am newbie in Linux.
thank you so much

**Comment by return42** [comment, marked as answer]
Close to lack of reply / hope your issue is solved / For a lively discussion its recommended to participate from the SearXNG community at [#searxng:matrix.org](https://matrix.to/#/#searxng:matrix.org).
